src/src-admin/Dashboard.js

```diff
@@ -16,7 +16,7 @@ const signout = () => {
   console.log("signout button clicked");
 
   sessionStorage.setItem('email',null);
-  window.location.href ='localhost:3000/admin';
+  window.location.replace('http://localhost:3000/admin');
 };
```

Comment:
A concern with referencing localhost:3000, when this is deployed, the url will be different, would it still work? To get the origin url (http://localhost:3000), you can do window.location.origin